### DISCUSSION TITLE
Subscribe to realtime changes in view collection

### DISCUSSION THREAD

**Discussion by Aragur** [pocketbase/pocketbase#2601] [Q&A]
Is it somehow possible to subscribe to real-time changes for a view collection?
I cannot find anything in the docs to do that.
If I subscribe via the SDK no events are fired.

**Comment by ganigeorgiev** [comment]
There are no realtime events for the view collections because they are only read-only, aka. they don't have create/update/delete actions.

**Reply by Aragur** [reply]
Thanks for the fast answer.
But to resources below there are these actions.
So is it somehow possible to get these events through?

**Reply by ganigeorgiev** [reply]
No, I don't think it is possible (at least not in a performant way) as the view query could be practically anything.
